src/app/api/_api-services/redis_service/index.ts

```
@@ -580,4 +585,31 @@ export class RedisService {
+		proposalType,
+		indexOrHash,
+		data
+	}: {
+		network: string;
+		proposalType: string;
+		indexOrHash: string;
+		data: IPostAnalytics;
+	}): Promise<void> {
+		const isActivePost = data.proposal.status && ACTIVE_PROPOSAL_STATUSES.includes(data.proposal.status as EProposalStatus);
+
+		await this.Set({
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)